config/opensearch_dashboards.yml

```diff
@@ -23,7 +23,7 @@
 #server.name: "your-hostname"
 
 # The URLs of the OpenSearch instances to use for all your queries.
-#opensearch.hosts: ["http://localhost:9200"]
+opensearch.hosts: ["http://localhost:9200"]
```

Comment:
Can you remove the changes made to this file? This is a boilerplate yaml config that should not have any vales set by default.

Reply:
Done